3-enrich/config/enrichments/iab_spiders_and_robots_enrichment.json

```diff
@@ -0,0 +1,26 @@
+{
+    "schema": "iglu:com.snowplowanalytics.snowplow.enrichments/iab_spiders_and_robots_enrichment/jsonschema/1-0-0",
+
+    "data": {
+
+        "name": "iab_spiders_and_robots_enrichment",
+        "vendor": "com.snowplowanalytics.snowplow.enrichments",
+        "enabled": true,
+        "parameters": {
+            "ipFile": {
+                "database": "ip_exclude_current_cidr.txt",
+                "uri": "s3://my-private-bucket/iab"
+            },
+            "excludeUseragentFile": {
+                "database": "exclude_current.txt",
+                "uri": "s3://my-private-bucket/iab"
+            },
+            "includeUseragentFile": {
+                "database": "include_current.txt",
+                "uri": "s3://my-private-bucket/iab"
+            },
+            "httpUsername": "USERNAME_PLACEHOLDER",
+            "httpPassword": "PASSWORD_PLACEHOLDER"
```

Comment:
nit: I'd remove basic auth or change the uris to be http to make the example coherent